docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1704,11 +1707,14 @@ you must configure the annotation processor in your build tool:
         <annotationProcessorPaths>
             <path>
                 <groupId>org.hibernate.orm</groupId>
-                <artifactId>hibernate-jpamodelgen</artifactId>
+                <artifactId>hibernate-processor</artifactId>
```

Comment:
We will probably need to write an OpenRewrite recipe for this one.

Reply:
Do we already have the recipes for this version? I've only seen some for the previous ones, when making the change but maybe I just missed it 🙈